suse_openstack_cloud

```diff
@@ -101,11 +101,16 @@ web_ui_admin_default_password "s3kr1t"
 .... "bmc_user"=>"root", "bmc_password"=>"cr0wBar!"}, {"token"=>"l33t", ...
 neutron_admin_password=s3kr1t
 rabbit_password=s3kr1t
-connection=postgresql://nova:s3kr1t@192.168.124.81/nova
+connection = mysql+pymysql://nova_api:s3kr1t@192.168.124.81/nova
```

Comment:
I think it would be nicer to keep the `postgresql://` URL and add the `mysql...` one as an additional line below it.  This will protect against regressions for Postgres.  I know it would probably be a weird situation if a SOC8 version of this plugin was running against Postgres (e.g. if something went wrong during an upgrade, before the DB migration happened?), but you never know - better safe than sorry ...

Reply:
works for me, but I fail to see how postgresql:// would be any different (more complex than mysql+pymsql://)

Reply:
@dirkmueller For example the `sed` pattern could be written with the assumption that the protocol contains two parts separated by `+`.  Even if that's not true now, in the future someone could decide to rewrite the sed pattern that way - the point is that the less the test code assumes about the implementation, and the more real-world test cases we provide, the more we're protected against potential future problems.  Yeah, now we're into bike-shedding realm, but I think the principle is important even if this particular case isn't life or death.

Reply:
@aspiers good point. I added the test coverage you requested, we don't have to argue about reducing test coverage you know ;)

could you please reevaluate?

Reply:
Done